### ISSUE TITLE
How to properly close RobustConnection?

### ISSUE THREAD

**Issue by kremius** [mosquito/aio-pika#218]
```python
import aio_pika
import asyncio

async def connect_to_rabbit():
    rabbit = await aio_pika.connect_robust('amqp://guest:guest@localhost:5672/')
    await rabbit.close()

loop = asyncio.get_event_loop()
for i in range(0, 100):
    loop.create_task(connect_to_rabbit())
loop.run_forever()
```
After ~10 seconds (reconnect interval I assume?) a lot of connections can be seen here  `http://127.0.0.1:15672/#/connections`:
![100-connections](https://user-images.githubusercontent.com/3407912/56949046-9135ad00-6b3a-11e9-883f-1a1457a020a7.png)
If `aio_pika.connect_robust` is replaced with simple `aio_pika.connect` then everything works as expected.
Is there any way to close robust connection reliably?

**Comment by zferentz** [comment]
experiencing the exact same behavior.
It seems that during the close() , the _on_connection_close is called (as expected) however this function also schedule the next reconnection. 

{code}
def _on_connection_close(self, connection, closing):
        self.connection = None

        # Have to remove non initialized channels
        self.__channels = {
            ch for ch in self.__channels if ch.number is not None
        }

        super()._on_connection_close(connection, closing)
        self.loop.call_later(
            self.reconnect_interval,
            lambda: self.loop.create_task(self.reconnect())
        )
{code}

I'd expect not to schedule the `self.reconnect` when i'm gracefully and explicitly closing the robust-connection. 

A possible solution might be to add a parameter to the `close()` method to disable reconnections however it'll break the nice interface. I wonder if there's a better way.

**Comment by mosquito** [comment]
@kremius Thank you for this issue.
@zferentz Thank you too for the investigation.

Fixes will be released with `aio-pika==5.5.3`.
